sahi/utils/tfhub.py

```diff
@@ -0,0 +1,102 @@
+def get_image(array):
+    import numpy as np
+    import tensorflow as tf
+
+    array = np.asarray(array, np.float32)
+    normalized_array = array
+    if array.max() <= 1:
+        normalized_array = array * 255.0
+
+    normalized_array = np.asarray(normalized_array, np.uint8)
```

Comment:
@kadirnar burada niye floattan inte ceviriyoruz goruntu formatini, ilgili bir tutorial/docs linki paylasabilir misin?

Reply:
[Tensorflow_hub](https://www.tensorflow.org/hub/tutorials/object_detection)
[Model Tutorial](https://tfhub.dev/tensorflow/ssd_mobilenet_v2/2)

2.link daha açıklayıcı

Reply:
hubdaki her modelin girdi ve cikti formatlari farkliymis yalniz: https://tfhub.dev/google/faster_rcnn/openimages_v4/inception_resnet_v2/1

senin implementasyonun hangi modelleri destekleyecek?

Reply:
https://tfhub.dev/s?module-type=image-object-detection&tf-version=tf2

Sadece tf2 modelleri destekleyeceğiz.

Reply:
attigin linkteki tf2 modellerinin cogunu desteklemiyor bizim su anki implementasyon.

ornegin attigin linkteki ilk model goruntu piksellerini 0-1 arasinda istiyor: https://tfhub.dev/rishit-dagli/yolo-cppe5/1

Reply:
```
array = np.asarray(array, np.float32)
img = (array - array.min()) / np.ptp(array)
img = np.asarray(img, np.float32)
img = tf.convert_to_tensor([img], tf.float32)
```
0-1 için bu kodu kullanabiliriz.

Reply:
0-1 veya 0-255 olup olmadığını kontrol edecek bir şey ayarlamaya çalışıyım

Reply:
Cozumu buldum, orijinal tensorflow modellerinin hepsi 0-256 aliyor, kullanimi onunla kisitlayabiliriz: https://tfhub.dev/tensorflow/collections/object_detection/1

model_path icinde `tfhub.dev/tensorflow` gecmiyorsa ValueError raise edebiliriz.